alf/environments/suite_carla.py

```diff
@@ -75,6 +75,10 @@ def is_available():
     return carla is not None
 
 
+MINIMUM_RENDER_WIDTH = 640
+MINIMUM_RENDER_HEIGHT = 240
```

Comment:
why changing from 480 to 240?

Reply:
The height of text panel is 240. There is no need beyond that.